api/v1alpha1/composable_types.go

```diff
@@ -74,6 +74,7 @@ type ComposableGetValueFrom struct {
 	Kind               string   `json:"kind"`
 	APIVersion         string   `json:"apiVersion"`
 	Name               string   `json:"name"`
+	Labels             []string `json:"labels"`
```

Comment:
Either `name` or `labels` should be defined. I don't know how to define mutual dependency, so probably we have to add `omnitemty` to both.

Reply:
okay, I will add `omni.temty`.